### DISCUSSION TITLE
SDXL Inpainting Fine Tune

### DISCUSSION THREAD

**Discussion by syedhaideralizaidi** [huggingface/diffusers#7060] [Q&A]
Do I need to include both the mask image and the output image in the zip folder for fine-tuning the SDXL Inpaint model, or is it sufficient to include only the output image? Is there any way we can fine tune SDXL Inpaint model?

**Comment by sayakpaul** [comment]
Cc: @patil-suraj 

A good reference: https://github.com/huggingface/diffusers/pull/6922 by @cryptexis
